### PR TITLE
Prepare for release 0.8

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- add new release presubmit jobs as per documentation
- add tests for release jobs

### CODE REVIEW

File: prow/jobs/kyma/components/application-proxy/application-proxy.yaml [hunk 1]
```diff
@@ -51,5 +51,16 @@ presubmits: # runs on PRs
     labels:
       <<: *job_labels_template
       preset-build-release: "true"
+  - name: pre-rel08-kyma-components-application-proxy
```
Comment (Szymongib) [inline prow/jobs/kyma/components/application-proxy/application-proxy.yaml]: As in the comment below: `Component no longer exists, do not create jobs for new releases`

File: prow/jobs/kyma/tools/load-test/load-test.yaml [hunk 2]
```diff
@@ -52,6 +52,16 @@ presubmits: # runs on PRs
       labels:
         <<: *job_labels_template
         preset-build-pr: "true"
+    - name: pre-rel08-kyma-tools-load-test
```
Comment (suleymanakbas91) [inline prow/jobs/kyma/tools/load-test/load-test.yaml]: This doesn't have `always_run: true`. Could you make sure if this was intended or a mistake?

File: prow/jobs/kyma/tools/failery/failery.yaml [hunk 3]
```diff
@@ -49,6 +49,16 @@ presubmits: # runs on PRs
       labels:
         <<: *job_labels_template
         preset-build-release: "true"
+    - name: pre-rel08-kyma-tools-failery
```
Comment (suleymanakbas91) [inline prow/jobs/kyma/tools/failery/failery.yaml]: This doesn't have `always_run: true`. Could you make sure if this was intended or a mistake?

File: prow/jobs/kyma/tools/docsbuilder/docsbuilder.yaml [hunk 4]
```diff
@@ -52,6 +52,16 @@ presubmits: # runs on PRs
       labels:
         <<: *job_labels_template
         preset-build-release: "true"
+    - name: pre-rel08-kyma-tools-docsbuilder
```
Comment (suleymanakbas91) [inline prow/jobs/kyma/tools/docsbuilder/docsbuilder.yaml]: This doesn't have `always_run: true`. Could you make sure if this was intended or a mistake?

File: prow/jobs/kyma/components/application-proxy/application-proxy.yaml (hunk 1 above)
Comment (k15r) [reply prow/jobs/kyma/components/application-proxy/application-proxy.yaml]: Sorry, totally missed that comment, the job has been removed

Comment (k15r) [reply prow/jobs/kyma/components/application-proxy/application-proxy.yaml]: it has been removed, i totally missed that comment

File: prow/jobs/kyma/tools/failery/failery.yaml (hunk 3 above)
Comment (k15r) [reply prow/jobs/kyma/tools/failery/failery.yaml]: forgotten
and fixed in next push

File: prow/jobs/kyma/tools/docsbuilder/docsbuilder.yaml (hunk 4 above)
Comment (k15r) [reply prow/jobs/kyma/tools/docsbuilder/docsbuilder.yaml]: forgotten
and fixed in next push

File: prow/jobs/kyma/tools/load-test/load-test.yaml (hunk 2 above)
Comment (k15r) [reply prow/jobs/kyma/tools/load-test/load-test.yaml]: this has been missed already in 07, i added it to the config

**Comment by michal-hudy** [comment]
Please add release jobs for `cms-controller-manager`

**Pull request merged**